### PR TITLE
Remove third party

### PR DESCRIPTION
**Pull request by laura-ding**
The packages of the third party are doing.

### CODE REVIEW

**Comment by laura-ding** [comment]
jenkins go

File: CMakeLists.txt [hunk 1]
```diff
@@ -142,6 +137,122 @@ if(NOT ${NEBULA_GPERF_BIN_DIR} STREQUAL "")
     list(INSERT CMAKE_PROGRAM_PATH 0 ${NEBULA_GPERF_BIN_DIR})
 endif()
 
+# third-party
+if(NOT ${NEBULA_BZIP2_ROOT} STREQUAL "")
+    message(STATUS "Specified NEBULA_BZIP2_ROOT: " ${NEBULA_BZIP2_ROOT})
+    list(INSERT CMAKE_INCLUDE_PATH 0 ${NEBULA_BZIP2_ROOT}/include)
+    list(INSERT CMAKE_LIBRARY_PATH 0 ${NEBULA_BZIP2_ROOT}/lib)
+    list(INSERT CMAKE_PROGRAM_PATH 0 ${NEBULA_BZIP2_ROOT}/bin)
+endif()
```
Comment (steppenwolfyuetong) [inline CMakeLists.txt]: Newly added options, please add a comment at the beginning of the file

File: cmake/FindFatal.cmake [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+#     find_package(Fatal)
+#
+# Variables used by this module, they can change the default behaviour and need
+# to be set before calling find_package:
+#
+# Variables defined by this module:
+#
+#  Fatal_FOUND            System has readline, include and lib dirs found
+#  Fatal_INCLUDE_DIR      The readline includes directories.
+#  Fatal_LIBRARY          The readline library.
```
Comment (steppenwolfyuetong) [inline cmake/FindFatal.cmake]: not use？

File: cmake/FindFbthrift.cmake [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+# Variables used by this module, they can change the default behaviour and need
+# to be set before calling find_package:
+#
+# Variables defined by this module:
+#
+#  Fbthrift_FOUND            System has fbthrift, thrift1 and include and lib dirs found.
+#  Fbthrift_INCLUDE_DIR      The fbthrift includes directories.
+#  Fbthrift_LIBRARY          The fbthrift library.
+#  Fbthrift_BIN              The fbthrift binary.
+#  Fbthrift_VERSION          The fbthrift version.
```
Comment (steppenwolfyuetong) [inline cmake/FindFbthrift.cmake]: not use？

File: CMakeLists.txt (hunk 1 above)
Comment (laura-ding) [reply CMakeLists.txt]: Good advice

**Comment by laura-ding** [comment]
jenkins go

**Comment by dutor** [comment]
Jenkins, please run tests.

**Comment by dutor** [comment]
> Unit testing failed.

``` /home/engshare/fbthrift/bin/thrift1: No such file or directory```

**Pull request merged**